core/src/test/java/com/linecorp/armeria/common/websocket/WebSocketFrameTypeTest.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2022 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package com.linecorp.armeria.common.websocket;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.junit.jupiter.api.Test;
+
+class WebSocketFrameTypeTest {
+
+    @Test
+    void controlFrame() {
+        assertThat(WebSocketFrameType.TEXT.isControlFrame()).isFalse();
+        assertThat(WebSocketFrameType.BINARY.isControlFrame()).isFalse();
+        assertThat(WebSocketFrameType.CLOSE.isControlFrame()).isTrue();
+        assertThat(WebSocketFrameType.PING.isControlFrame()).isTrue();
+        assertThat(WebSocketFrameType.PONG.isControlFrame()).isTrue();
+    }
```

Comment:
Let's ensure we didn't miss out any enums by adding an assertion that makes sure there are no other enums.

Reply:
Changed to use `ParameterizedTest`. 😉